### ISSUE TITLE
Port scanner

### ISSUE THREAD

**Issue by BBBB98** [pi-node/instructions#159]

**Comment by ihatejam** [comment]
You've entered your internal IP address -- ipfingerprints cannot check that. 
When you load the page on ipfingerprints, it will automatically fill in your public IP address (do not change it). Just update the port section start = 31400 and end 31409.

Please do not post your public IP address in this forum.

I believe that I have already mentioned in a different thread that your setup is correct. All you have to do is wait for your PC to be recognized on the testnet network as a node candidate.

**Comment by BBBB98** [comment]
> You've entered your internal IP address -- ipfingerprints cannot check that.
> When you load the page on ipfingerprints, it will automatically fill in your public IP address (do not change it). Just update the port section start = 31400 and end 31409.
> 
> Please do not post your public IP address in this forum.
> 
> I believe that I have already mentioned in a different thread that your setup is correct. All you have to do is wait for your PC to be recognized on the testnet network as a node candidate.

Sir how do I delete my issues

**Comment by ihatejam** [comment]
You don't need to delete them - you have not posted anything bad. Apologies if I've made you think so. You can just mark your issues as closed.

**Comment by BBBB98** [comment]
> You don't need to delete them - you have not posted anything bad. Apologies if I've made you think so. You can just mark your issues as closed.

No, it's OK. Because u say that not to post ip address in this forum so I want to delete.

**Comment by ihatejam** [comment]
On this occasion, you were fine - because you posted your **internal** IP address which nobody can reach from outside of your home network (which is why ipfingerprints showed it as closed). 
Your **external** IP address (bound to your router) also known as your **public** IP address is the one to look out for and never share.

I hope that helps.
